projects.json

```diff
@@ -2963,6 +2963,31 @@
       }
     ]
   },
+  {
+    "repository": "Git",
+    "url": "https://github.com/apple/swift-numerics",
+    "path": "swift-numerics",
+    "branch": "main",
+    "maintainer": "stephentyrone@gmail.com",
```

Comment:
Is there a reason this is personal email vs apple email?

Reply:
Just using the one listed on GitHub account.

@stephentyrone Do you have a preference for which email is listed here?